tests/app/celery/test_process_delivery_status_result_tasks.py

```diff
@@ -538,6 +550,392 @@ def test_get_include_payload_status_exception(notify_api, mocker, sample_notific
     assert not _get_include_payload_status(sample_notification())
 
 
+@pytest.mark.parametrize('exception', [ConnectionError, RedisError, ResponseError, TimeoutError])
+def test_update_sms_retry_count_redis_set_exception(mocker, exception):
+    mocker.patch(
+        'app.celery.process_delivery_status_result_tasks.redis_store.set',
+        side_effect=exception,
+    )
+
+    with pytest.raises(Exception) as e:
```

Comment:
Usually we're looking for the specific exception here, but I won't block it for that.